Add July 2023 newsletter (#1981)
writethedocs/www: docs/blog/newsletter-july-2023.rst

.. post:: July 07, 2023
   :tags: newsletter

#####################################
Write the Docs Newsletter – July 2023
#####################################

Aloha! There's lots to cover this month, so let's dive in.

> Who makes up the Write the Docs community? What proportion support pay transparency?  Where are salaries highest? How many have started a new job in the past 12 months? All these questions and more are answered in the `2022 Write the Docs Salary Survey results <https://writethedocs.org/surveys>`__. Stay tuned: the survey for 2023 will be opening in early September.

Reviewer: ```suggestion
Who makes up the Write the Docs community? What proportion support pay transparency? Where are salaries highest? How many have started a new job in the past 12 months? All these questions and more are answered in the `2022 Write the Docs Salary Survey results <https://writethedocs.org/surveys>`__. Stay tuned: the survey for 2023 will be opening in early September.
```

> In conference news, we recently saw the announcement of the `speakers for the 2023 Atlantic conference </conf/atlantic/2023/news/announcing-speakers/>`__. So check out all the interesting things you can look forward to learning. And the `Australia conference </conf/australia/2023/news/welcome/>`__ is getting ready for the call for proposals and ticket sales so stand by for more about that.

Author: ```suggestion
In conference news, we recently saw the announcement of the `speakers for the 2023 Atlantic conference </conf/atlantic/2023/news/announcing-speakers/>`__. So check out all the interesting things you can look forward to learning. And the Australia conference has released the `call for proposals </conf/australia/2023/cfp/>`__ and `ticket sales </conf/australia/2023/tickets/>`__ so get ready for good times in December.
```

------------------------
Getting Info from Others
------------------------

A recent thread resurrected an enduring issue with documentarians: How do we get the information we need from experts? The `May 2020 newsletter </blog/newsletter-may-2020/#getting-more-info-from-smes>`__ addressed this issue. The current discussion focused on how to ask questions of developers: what questions to ask and how to get the best responses for documentation. 

People indicated that there are two levels of questions: 1) general, high-level questions; and 2) specific, detailed questions. Documentarians ask general questions to get an overall understanding of a product or domain. After we have the higher-level understanding, we can determine what detailed questions to ask. Specific questions fill in the gaps in our knowledge.

> For general questions, you can ask them of product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.

Reviewer: In the first sentence, maybe "For general questions, you can ask product managers..." (remove the `them of`)?

Author: Sounds better to me.

```suggestion
For general questions, you can ask product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.
```
